src/Common/Twig/TwigExtension.php

```diff
@@ -203,6 +203,13 @@ function ($string) {
                 function ($string) {
                     return xl_document_category($string);
                 }
+            ),
+
+            new TwigFilter(
+                'xlFormTitle',
+                function ($string, $mode = 'r', $prepend = '', $append = '') {
+                    return xl_form_title($string, $mode, $prepend, $append);
```

Comment:
those last 3 parameters are not needed (legacy stuff), and would remove them in above 2 lines.